styles.css

```diff
@@ -1,146 +1,296 @@
+header {
+  background: #fff;
+}
+
+li {
+  list-style: none;
+}
+
+.mobile-portfolio {
+  position: absolute;
+  left: 0;
+  right: 0;
   display: flex;
-  flex-direction: row;
+  width: 100%;
+  height: 50px;
+  justify-content: space-between;
   align-items: center;
-  padding: 10px 12px;
-  position: absolute;
-  width: 100px;
-  height: 40px;
-  left: 12px;
-  top: calc(50% - 40px / 2);
+  padding: 14px 24px;
 }
 
-.logo a {
-  width: 76px;
-  height: 20px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
+.logo {
   font-weight: 700;
   font-size: 18px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
   text-align: center;
   color: #6070ff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
-.menu {
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  padding: 8px;
-  position: absolute;
-  width: 40px;
-  height: 40px;
-  right: 24px;
-  top: calc(50% - 40px / 2);
-  border-radius: 8px;
-}
-
-.bx {
-  position: relative;
-  right: 10.67%;
-  top: 32%;
-  bottom: 50%;
-  font-size: 30px;
-}
-
-.headline {
-  background-image: url("header-shapes mobile.svg");
+.first-page {
+  min-width: 375px;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  background: #fff;
+  background-image: url("./image/bg.svg");
   background-repeat: no-repeat;
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  padding: 114px 24px;
-  gap: 12px;
-  position: absolute;
-  width: 375px;
-  height: 604px;
-  left: calc(50% - 375px / 2);
-  top: 60px;
+  background-size: calc(100%);
 }
 
 .name {
-  width: 327px;
-  height: 150px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
+  font-weight: 900;
+  font-size: 50px;
   line-height: 52px;
   display: flex;
   align-items: center;
   color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
 .text {
-  width: 327px;
-  height: 146px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+  margin-right: 10px;
+  background: none;
 }
 
-.media {
-  width: 130px;
-  height: 24px;
+.connect {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
+  text-decoration: none;
+  margin-top: 30px;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+  margin-left: -40px;
+}
+
+.work {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 10px;
+}
+
+.project {
+  background: #fff;
+  border: 0.2rem solid #e5e5e5;
+  border-radius: 20px;
+  flex-direction: row;
+}
+
+.project:last-child {
+  margin-bottom: 40px;
+}
+
+.project-img {
+  width: 265px;
+  height: 220px;
+  flex-direction: row;
   flex: none;
-  order: 0;
   flex-grow: 0;
+  margin-left: 30px;
+  margin-top: 20px;
 }
 
```

Comment:
- [ ]  Good job implementing flex box as suggested by reviewer. However, I think think there are still few steps to go to implement it properly so that you can have the design according to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Z5hPenaSYnDkoWHJ-0) I know you can do that 😃💪

### Your design previously 🔽
![image](https://user-images.githubusercontent.com/106119105/212491564-a717b228-e31c-440e-995b-75f6c7511669.png)

### Your design now 🔽
![image](https://user-images.githubusercontent.com/106119105/212491576-7a706e8a-cc9e-4bf1-972f-5645295aa2ba.png)

### figma template 🔽
![image](https://user-images.githubusercontent.com/106119105/212491557-0bc61565-71ed-4bdf-b543-6d7980fb1365.png)

- [ ] In your `project class` you are using `flex-direction: row` property but I probably you forgot to use `display: flex` as suggested earlier.  Please note that `flex-direction: row` is the default property of `flex`. [Check out this for more clarity](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) 🔗. As an example, you can use something like below 👇

```
.project {
    // other properties
    display: flex;
    flex-direction: column;
}

.project-img {
    width: 100%;
    padding: 15px;
    margin: 0;
}
```